### ISSUE TITLE
[Brazilian Portuguese] Accessibility Principles

### ISSUE THREAD

**Issue by adielcristo** [w3c/wai-translations#257]
### Page URL

https://www.w3.org/WAI/fundamentals/accessibility-principles/

### Instructions

- [X] I have read the information on [Translating WAI Documents](https://www.w3.org/WAI/about/translating/)
- [X] Before starting the translation, I will wait for confirmation that the resource is ready for translation.

### Link to original file in GitHub

https://github.com/w3c/wai-website/blob/main/pages/fundamentals/accessibility-principles/index.md

**Comment by remibetin** [comment]
Hi @adielcristo,

May I suggest to postpone this translation? Indeed, this page contains a lot of wording from WCAG 2 and a [Brazilian Portuguese Authorized translation of WCAG 2.2](https://www.w3.org/WAI/standards-guidelines/wcag/translations/#pt-BR22) is still in-progress.

**Comment by adielcristo** [comment]
Hi @remibetin,

Sure. Meanwhile, I'll continue to work on the Accessibility Fundamentals section.
